dex/router/src/config.rs

```diff
@@ -0,0 +1,69 @@
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+use crate::{enable_swap_by_user::EnableSwapByUserConfig, factory::PairTokens};
+
+#[multiversx_sc::module]
+pub trait ConfigModule {
+    fn is_active(&self) -> bool {
+        self.state().get()
```

Comment:
Couldn't you have use the pausable module?

Reply:
The same as before, this was the code from the beginning and I would not change it because of backwards compatibility.